src/main/java/pl/psnc/dei/ui/components/imports/CreateImportComponent.java

```diff
@@ -178,12 +180,12 @@ private Component generateSwitchingButtons() {
 			refresh();
 		});
 		switchingButtons.add(moveFromSelectedToAll);
-		moveFromSelectedToAll.setEnabled(shouldBeReadOnly());
+		moveFromSelectedToAll.setEnabled(!shouldBeReadOnly());
 		return switchingButtons;
 	}
 
 	private boolean shouldBeReadOnly() {
```

Comment:
Since method result is always inverted, you can probably just change the condition inside and rename the method